report_thesis/src/sections/problem_definition.tex

```diff
@@ -0,0 +1,54 @@
+\section{Problem Definition}
+\label{sec:problem_definition}
+
+Our research aims to improve the accuracy and robustness of major oxide predictions derived from \gls{libs} data, building upon the baseline established in \cite{p9_paper}.
+There are many challenges in predicting major oxides from \gls{libs} data, including the high dimensionality and non-linearity of the data, as well as the presence of multicollinearity.
+Some of these are caused by \textit{matrix effects}\cite{andersonImprovedAccuracyQuantitative2017}, which is a catch-all term for any effect that can cause the intensity of emission lines from an element to vary, independent of that element's concentration. So it's unknown variables that affect the results.
+Furthermore, due to the high cost of data collection, datasets are often small, which further complicates the task of building accurate and robust models.
+
+Based on the limitations with the current MOC pipeline, as reported in \cite{p9_paper}, we identified three key areas for further investigation: dimensionality reduction, model selection, and outlier removal.
+
+In this work, we focus on dimensionality reduction and model selection over outlier removal.
+This is justified by the low incidence of outliers in the ChemCam \gls{libs} calibration dataset, as reported in \cite{p9_paper}.
+Dimensionality reduction is crucial for managing the high-dimensional nature of \gls{libs} data. % TODO: There are lots of related works which explore DR in LIBS data. We can back this up with citations.
+Furthermore, model selection shows promise in addressing the limitations of the current MOC pipeline, as it allows for the exploration of a wider range of algorithms, potentially leading to improved performance.
+We showed that advanced ensemble methods, such as \gls{gbr}, and deep learning models, such as \gls{ann}, have the potential to outperform the current MOC pipeline.
+Methods are selected based on their promise in handling high-dimensional, non-linear data. Ideally, the selected methods should also be feasible for small datasets, a common scenario in \gls{libs} analyses. 
```

Comment:
Their "promise"? What does that mean? Do you mean "potential to handle high-dimensional, non-linear data"?

Reply:
Promise can be used analogously to potential in this case.